doc/Makefile

```diff
@@ -38,14 +38,12 @@ help:
 clean:
 	-rm -rf $(BUILDDIR)/*
 
-html:
-	@echo
-	@echo "Install dependencies"
-	$(PIP) install furo
-	@echo "Finished installing dependencies"
+build-html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	@echo "HTML build complete"
+
+html: build-html
```

Comment:
Why split this?

Reply:
I opened  #6623 I wanted to remove the commit from this branch.